src/os/ui/im/action/filteractionimporter.js

```diff
@@ -64,6 +64,17 @@ os.ui.im.action.FilterActionImporter.prototype.getFilterTooltip = function(entry
 };
 
 
+/**
+ * @param {os.layer.Vector} vectorLayer
+ * @return {Array}
+ * @override
+ * @suppress {checkTypes}
```

Comment:
Using this suppression is typically a big red flag. Let's figure out the proper typing to make the compiler happy instead.